Support multiple orgs on home screen

jonahkagan opened the pull request:
Also did some redesign of the home and login screens while I was at it. I mostly just reorganized the code from `CreateAudit` into a few different components.

I also found a bug where the audit list doesn't reload after you create a new audit, so I fixed that as well.

<img width="539" alt="Screen Shot 2020-08-31 at 2 47 58 PM" src="https://user-images.githubusercontent.com/530106/91778647-02aac300-eba8-11ea-8f81-e9196ac70eab.png">

![image](https://user-images.githubusercontent.com/530106/91778698-2837cc80-eba8-11ea-8fd7-fd0f190a0549.png)

<img width="1024" alt="Screen Shot 2020-09-03 at 11 53 01 AM" src="https://user-images.githubusercontent.com/530106/92155312-0b83da80-eddc-11ea-9618-dfb8a7febf03.png">

jonahkagan commented:
@MorganLove would love to know your initial thoughts before I start in on the tests

MorganLove commented:
![Screenshot from 2020-09-01 08-16-29](https://user-images.githubusercontent.com/8096043/91870253-ab4e3680-ec2b-11ea-87d7-cb471ee6bd80.png)

@jonahkagan I love the look on the login screen and the audit admin screen! I think the jurisdiction admin screen got a bit messed up in the process though.

MorganLove commented on client/src/components/HomeScreen.tsx:
```diff
@@ -0,0 +1,265 @@
+}
+
+const ListAuditsWrapper = styled.div`
+  padding: 30px;
```
```suggestion
  position: relative;
  width: 100%;
  padding: 30px;
```

MorganLove commented on client/src/components/HomeScreen.tsx:
```diff
@@ -0,0 +1,265 @@
+const ListAuditsWrapper = styled.div`
+  padding: 30px;
+`
+
```
```suggestion

const JurisdictionAuditsWrapper = styled.div`
  display: inline-block;
  position: absolute;
  margin: 50px;
  left: 50%;
  transform: translate(-50%, -50%);
  text-align: center;
`

```

MorganLove commented on client/src/components/HomeScreen.tsx:
```diff
@@ -0,0 +1,265 @@
+            )}
+          </div>
+        ))}
+      {meta.type === 'jurisdiction_admin' &&
+        meta.jurisdictions.map(({ id, name, election }) => (
+          <AuditLink key={id} to={`/election/${election.id}`} intent="primary">
+            {name} - {election.auditName}
+          </AuditLink>
+        ))}
```
```suggestion
      {meta.type === 'jurisdiction_admin' && (
        <JurisdictionAuditsWrapper>
          <h2>Audits</h2>
          {meta.jurisdictions.map(({ id, name, election }) => (
            <AuditLink
              key={id}
              to={`/election/${election.id}`}
              intent="primary"
            >
              {name} - {election.auditName}
            </AuditLink>
          ))}
        </JurisdictionAuditsWrapper>
      )}
```

jonahkagan commented:
Fixed the jurisdiction view and added a screenshot. It got a bit complicated because a JA can be the admin for multiple jurisdictions within an audit or across multiple audits.

jonahkagan commented:
@MorganLove I fixed up the JA screen. Decided not to center it so that it's consistent with the rest of the app (plus a designer once told me I should never center anything except a login screen, and it's been pretty good advice)

Also found a bug where the audit list doesn't refresh after you create a new audit. Came up with a quick slightly hacky solution, which I commented in the code. Let me know your thoughts. I didn't want to try to do a larger-scale solution for now

jonahkagan commented:
Also added some initial tests

MorganLove commented:
Looking great!

jonahkagan commented:
@MorganLove ready for review!
